### PR TITLE
Creating DDL

### PR DESCRIPTION
**Pull request by HongtingYin**
https://www.notion.so/417963db3cdf4ac0b4c36a9b72950b2c?v=88fd7654e97c46cc9c81cbd37307123e&p=c59d1985087140ea99d6bac3a2e3e24d

### CODE REVIEW

File: linux_sql/scripts/psql_docker.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	exit 0
+else
+	echo "Error!"
+	echo "usage:./psql_docker.sh start|stop password"
+	exit 1
+
+fi
+
+exit 0
+
```
Comment (syumet) [inline linux_sql/scripts/psql_docker.sh]: `psql_docker.sh` is another feature. In this project, we should create the branch `feature/create_database` and open a separate PR.

**Comment by syumet** [comment]
The same for `host_info.sh` and `host_usage.sh`, they belong to `feature/agent`.

**Comment by syumet** [comment]
Please use more meaningful commit messages.

**Comment by HongtingYin** [comment]
Hey Steven, I have fixed all branches now, please review them, thanks!

**Pull request merged**